### PR TITLE
CI Update - .NET 7 Preview 3 APIs

### PR DESCRIPTION
**Pull request by gewarren**
Build.Reason:Manual by Genevieve Warren
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=289998&view=results
 source_repo.branch:net7-pre3
 source_repo.url:https://apidrop.visualstudio.com/_git/binaries

### CODE REVIEW

**Review by rzikm — APPROVED** [review]
System.Net.* parts LGTM!

File: xml/System.Formats.Cbor/CborConformanceMode.xml [hunk 1]
```diff
@@ -42,9 +42,15 @@
           <para>Ensures that the CBOR data is canonical, as specified in RFC7049 section 3.9.</para>
           <para>Extends strict conformance with the following requirements:</para>
           <list type="bullet">
-            <item>Integers must be encoded as small as possible.</item>
-            <item>Maps (major type 5) must contain keys sorted by encoding.</item>
-            <item>Indefinite-length items must be made into definite-length items.</item>
+            <item>
+              <description>Integers must be encoded as small as possible.</description>
```
Comment (gewarren) [inline xml/System.Formats.Cbor/CborConformanceMode.xml:46]: Check this in the preview.

File: xml/System.Formats.Cbor/CborReader.xml [hunk 2]
```diff
@@ -197,11 +197,13 @@
         <returns>The decoded value.</returns>
         <remarks>Useful in scenarios where the semantic value decoder needs to be determined at run time.</remarks>
         <exception cref="T:System.InvalidOperationException">The next data item does not have the correct major type.</exception>
-        <exception cref="T:System.Formats.Cbor.CborContentException">The next value has an invalid CBOR encoding.
-            -or-
-            There was an unexpected end of CBOR encoding data.
-            -or-
-            The next value uses a CBOR encoding that is not valid under the current conformance mode.</exception>
+        <exception cref="T:System.Formats.Cbor.CborContentException">
+          <para>The next value has an invalid CBOR encoding.</para>
```
Comment (gewarren) [inline xml/System.Formats.Cbor/CborReader.xml:201]: Check in preview.

File: xml/System.Data.Odbc/OdbcPermissionAttribute.xml [hunk 3]
```diff
@@ -30,7 +30,7 @@
       <AttributeName Language="C#">[System.AttributeUsage(System.AttributeTargets.Assembly | System.AttributeTargets.Class | System.AttributeTargets.Constructor | System.AttributeTargets.Method | System.AttributeTargets.Struct, AllowMultiple=true, Inherited=false)]</AttributeName>
       <AttributeName Language="F#">[&lt;System.AttributeUsage(System.AttributeTargets.Assembly | System.AttributeTargets.Class | System.AttributeTargets.Constructor | System.AttributeTargets.Method | System.AttributeTargets.Struct, AllowMultiple=true, Inherited=false)&gt;]</AttributeName>
     </Attribute>
-    <Attribute FrameworkAlternate="dotnet-plat-ext-5.0;windowsdesktop-5.0;windowsdesktop-6.0;windowsdesktop-7.0">
+    <Attribute FrameworkAlternate="dotnet-plat-ext-5.0;dotnet-plat-ext-7.0;windowsdesktop-5.0;windowsdesktop-6.0;windowsdesktop-7.0">
```
[the remark this answers is not in the document]
Comment (carlossanlop) [reply xml/System.Data.Odbc/OdbcPermissionAttribute.xml:33]: Why were you expecting that moniker for this API specifically?

I assume the contents of this PR are generated using mdoc. Do we have control over which monikers to include? For example, the Continuous Integration job for Intellisense has a textbox where we can manually indicate the monikers we want to include:

image

**Review by carlossanlop — APPROVED** [review]

File: xml/System.Data.Odbc/OdbcPermissionAttribute.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (gewarren) [reply xml/System.Data.Odbc/OdbcPermissionAttribute.xml:33]: It's weird that this type would show as obsolete in .NET 5 and .NET 7, but not .NET 6. So you won't see the CAUTION block for the obsoletion if you choose the .NET 6 moniker: https://review.docs.microsoft.com/en-us/dotnet/api/system.data.odbc.odbcpermissionattribute?view=dotnet-plat-ext-6.0&branch=pr-en-us-8000.

But it looks like the attribute was only [added in src in February](https://github.com/dotnet/runtime/pull/65847), so this is correct.

**Pull request merged**